File: siserp-logistica-backend/src/main/java/com/besoft/siserp/lgtc/dto/response/ProveedorJuridRepLegalListarResponse.java
package com.besoft.siserp.lgtc.dto.response;

import java.io.Serializable;
import java.util.Date;

import lombok.Getter;
import lombok.Setter;
import lombok.ToString;

@Getter
@Setter
@ToString
public class ProveedorJuridRepLegalListarResponse implements Serializable {

	private static final long serialVersionUID = 1L;

	private Long id;
	private Long idProveedor;
	private Long idtTipoDocumento;
	private String nomTipoDocumento;
	private String nroDocumento;
	private Long idtGenero;
	private String nomGenero;
	private Long idtEstadoCivil;
	private String nomEstadoCivil;
	private String nombre;
	private String apellidoPat;
	private String apellidoMat;
	private String cargo;
	private String telefono;
	private String email;
	private String codUbigeo;
	private String direccion;
	private Date fecNacimiento;
	private Integer flagActivo;
	private Long idUsuarioCrea;
	private Date fecUsuarioCrea;
	private Long idUsuarioMod;
	private Date fecUsuarioMod;

}

File: siserp-logistica-frontend/src/main/siserp-logistica-frontend/src/app/modules/intranet/dto/response/proveedor-jurid-rep-legal-listar.response.ts
export class ProveedorJuridRepLegalListarResponse {
    id: number;
    idProveedor: number;
    idtTipoDocumento: number;
    nomTipoDocumento: string;
    nroDocumento: string;
    idtGenero: number;//ADICIONAR
    nomGenero: number;//ADICIONAR
    idtEstadoCivil: number;//ADICIONAR
    nomEstadoCivil: number;//ADICIONAR
    nombre: string;
    apellidoPat: string;
    apellidoMat: string;
    cargo: string;
    telefono: string;
    email: string;
    codUbigeo: string;
    direccion: string;//ADICIONAR
    fecNacimiento: Date;//ADICIONAR
    flagActivo: number;
    idUsuarioCrea: number;
    fecUsuarioCrea: Date;
    idUsuarioMod: number;
    fecUsuarioMod: Date;
}